flow2ml/models/model.py

```diff
@@ -0,0 +1,29 @@
+import pandas as pd
+import numpy as np
+from sklearn.linear_model import LogisticRegression
+from sklearn.model_selection import GridSearchCV
+import os
+
+class Apply_Models:
```

Comment:
Please change this to ApplyModels(classnames should not contain underscores as per code standards)